File: program/program/find-lcm-of-two-numbers/FindLcmOfTwoNumbers.java
import java.util.Scanner;

public class findlcmoftwonumbers {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Input : ");
        int num1 = input.nextInt();
        int num2 = input.nextInt();
        int min = 0 ;
        int max = 0;
        if (num1<num2) {
            min = num1;
            max = num2;
        }
        else {
            min = num2;
            max = num1;
        }
        for (int i = 1; i < max+1; i++) {
            if ((min*i)%max==0) {
                System.out.println("Output : "+min*i);
                break;
            }

        }
    }
}

File: program/program/find-lcm-of-two-numbers/find_lcm_of_two_numbers.go
package main

import (
	"fmt"
)

func gcd(a,b int) int{
    if b == 0 {
        return a
    }
    return gcd(b,a%b)
}
func main(){
    var num1, num2 int
    fmt.Print("Input 1st number for LCM : ")
    fmt.Scanln(&num1)
    fmt.Print("Input 2nd number for LCM : ")
    fmt.Scanln(&num2)

    lcm := (num1*num2)/gcd(num1,num2)
    fmt.Print("Output : ",lcm)
}